README.md

```diff
@@ -8,7 +8,15 @@ However, as it seemed to be discontinued, I decided to rework it entirely and ad
 
 ## Install
 The mod works through the [Hollow Knight Modding API](https://github.com/seresharp/HollowKnight.Modding) (a getting started guide can be found [here](https://radiance.host/apidocs/Getting-Started.html)).
-After installing the API, this mod can be installed by dropping the compiled DLL into your mods folder, which can be found in your Steam installation: `<steam>/Hollow Knight/hollow_knight_Data/Managed/Mods/`.
+After installing the API, this mod can be installed by dropping the compiled DLL into your mods folder, which can be found in your Steam installation:
+
+- **Windows**: `C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight\hollow_knight_Data\Managed\Mods\`.
```

Comment:
Perhaps a mention that this is only the case if Hollow Knight is installed to the default steam location. Moreover, the aforementioned doesn't hold when Steam is installed on a differently named volume (e.g. `D:\`).
Another option would be to abstract away from that and call it the Steam install folder, although this is less straightforward for less tech-savvy people.

Reply:
hmm afaik the only difference for a D:\ drive would be the letter at the start. not sure tho